modules/core/src/main/java/org/apache/ignite/internal/processors/cache/CacheOperationContext.java

```diff
@@ -31,10 +32,17 @@
  * Cache operation context.
  */
 public class CacheOperationContext implements Serializable {
-    /** */
-    //TODO IGNITE-8801 remove this and set default as `false`.
-    public static final boolean DFLT_ALLOW_ATOMIC_OPS_IN_TX =
-        IgniteSystemProperties.getBoolean(IGNITE_ALLOW_ATOMIC_OPS_IN_TX, true);
+    /**
+     * Since 2.15.0 atomic operations inside transactions are not allowed.
+     * To return the previous behaviour and to allow transactions in operations with atomic caches you can set system property
+     * {@link IgniteSystemProperties#IGNITE_ALLOW_ATOMIC_OPS_IN_TX IGNITE_ALLOW_ATOMIC_OPS_IN_TX} to {@code true}.
+     * <p>
+     * If you want to use atomic operations inside transactions in case they are restricted by system property,
+     * you should allow it before transaction start.
+     */
```

Comment:
no need such a doc here, it's enough to have it only in `IgniteCache`.

Reply:
thank you, I will delete